srunner/scenariomanager/actorcontrols/actor_control.py

```diff
@@ -21,7 +21,7 @@
 import carla
 
 from srunner.scenariomanager.actorcontrols.external_control import ExternalControl
-from srunner.scenariomanager.actorcontrols.npc_vehicle_control import NpcVehicleControl
+from srunner.scenariomanager.actorcontrols.simple_vehicle_control import SimpleVehicleControl
```

Comment:
Will this simple controller affect any other scenarios? I.e. will it always work as intended?

Reply:
The problem is that none of the controllers are optimal but this simple controller is the better of the two imo. The previous controller uses a PID controller to steer the vehicle both lateral and longitudinal which results in some response time (suboptimal for a scenariorunner imo). The new controller sets the velocity and angular velocity of the vehicle directly and thus makes the ScenarioRunner focus more on the scenario itself rather than following a trajectory to the best of its ability. The PID controller might be more "realistic" in a sense, but demands a lot of tuning to work half as good as the real controllers in a car. So in the end it's not really more realistic anyway.